locksmith/migrations/20240522155101-Created userAccount.js

```diff
@@ -0,0 +1,42 @@
+'use strict'
+
+/** @type {import('sequelize-cli').Migration} */
+module.exports = {
+  async up(queryInterface, Sequelize) {
+    await queryInterface.createTable('UserAccounts', {
+      id: {
+        allowNull: false,
+        primaryKey: true,
+        type: Sequelize.UUID,
+        defaultValue: Sequelize.UUIDV4,
+      },
+      emailAddress: {
+        type: Sequelize.STRING,
+        allowNull: false,
+        unique: true,
+      },
+      loginMethod: {
```

Comment:
Let's make this an array please. I can imagine a world where we might have a user with multiple methods in the future.